src/sass/extends/_global-header.scss

```diff
@@ -931,6 +951,6 @@ $fa-search: url($image-assets-path + '/font-awesome-svg/search.svg');
   #asu-header.scrolled + div,
   #asu-header.scrolled + main {
     @include transition;
-    margin-top: 81x;
+    margin-top: 81px;
   }
 }
```

Comment:
The component header that is launched by the asu_brand module provides its own styles.  Are these in _global-header.scss necessary? I see it's just the scss, so perhaps it doesn't get included in the styles that are bundled. Just curious. If they are being included, they may not need to be, and excluding them would help us slim down our CSS.

Reply:
Yes they are being included. I had thought about this but not found time to test removing them. I'll do that at some point today.